HIP-UserStore.Model/HIP-UserStore.Model.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
For better compatibility, this project should target .NET Standard (e.g. "netstandard1.5")